CONTRIBUTING.md

```diff
@@ -0,0 +1,234 @@
+# How to contribute model to Open Model Zoo
+
+From this document you will learn how to contribute your model to OpenVINO&trade; Open Model Zoo (OMZ). It could be done in few steps.
+
+1. [Model location](#model-location)
+2. [Model conversion](#model-conversion)
+3. [Demo](#demo)
+4. [Accuracy validation](#accuracy-validation)
+7. [Configuration file](#configuration-file)
+6. [Documentation](#documentation)
+7. [Pull request requirements](#pull-request-requirements)
+
+List of supported frameworks:
+* Caffe\*
+* Caffe2\* (by conversion to ONNX\*)
+* TensorFlow\*
+* MXNet\*
+* PyTorch\* (by conversion to ONNX\*)
+
+## Model location
+
+Upload your model to any Internet file storage. The main requirements are that the model must either be downloadable from a direct HTTP(S) link or from Google Drive\*.
+
+*After this step you will get **links** to the model*
+
+## Model conversion
+
+Deep Learning Inference Engine (IE) supports models in Intermediate Representation (IR) format. Model from any supported framework can be  converted to IR using Model Optimizer tool included in OpenVINO&trade; package. More information about conversion you can learn [here](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html). After successful conversion you will get model in IR format `*.xml` representing net graph and `*.bin` containing net parameters. 
```

Comment:
...`A` model from any supported framework can be `extra space`converted...

`More information about conversion you can learn [here]`->`Find more information about conversion in [Model Optimizer Developer Guide]`

Reply:
There is an extra bracket now: `[[Model Optimizer Developer Guide]`

Reply:
Thanks, fixed!